app/assets/javascripts/index.js

```diff
@@ -0,0 +1,48 @@
+$(() => {
+    fetchIdeas()
+})
+
+var fetchIdeas = () => {
+    $.getJSON("/api/v1/ideas", function(data) {
+        chronoSortIdeas(data)
+    })
```

Comment:
This could be shortened to `$.getJSON('/api/v1/ideas', chronoSortIdeas)`.

Alternatively, you could just have Rails pull the ideas out of the database in chronological order.